department-of-veterans-affairs/va.gov-cms
# VACMS-7092: Cleanup node module permissions

- Updated "Node" permissions for various roles.
- Updated PHPUnit permissions for various roles.
- More permissions adjustments for various roles. Removed "delete any content" permission from role.
- Updated permissions in PHPUnit for various roles.

## Description

closes #7092

## Testing done

All tests passes

## Screenshots


## QA steps

As user administrator
1. Go to `/admin/people/permissions` to verify and confirm that:
    - [ ] Only "Content creator" roles should have Create permissions. Editor / Reviewer / Publisher roles should not.
    - [ ] No role should have Delete content permission checked
    - [ ] No role should have Delete revisions content permission checked
    - [ ] All content types should be editable by all users
    - [ ] No role should have any permission related to "edit own" or "delete own" permissions
    - [ ] Content publisher role should have "Revert revisions" permission for all content types
    - [ ] All roles should have the View revisions permission checked for all content types except _Knowledge Base_

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
